### PR TITLE
Search Bar

### PR DESCRIPTION
**Pull request by JoachimTerepo**
Just created a Search Bar, just confirming if we are doing BibTex Format or not. But this is what I have completed

### CODE REVIEW

File: frontend/src/app/globals.css
Comment (TheRangiCrew) [inline frontend/src/app/globals.css:1]: DO NOT DELETE THE TAILWIND REFERENCES

File: frontend/src/components/SearchBar.tsx
Comment (TheRangiCrew) [inline frontend/src/components/SearchBar.tsx:1]: Are these methods hardcoded?? The database has claims and methods which can be dynamically loaded like they should be.
Check `main`

File: frontend/src/styles/globals.css
Comment (TheRangiCrew) [inline frontend/src/styles/globals.css:1]: This file seems like a duplicate of `/app/global.css`???

**Review by TheRangiCrew — CHANGES_REQUESTED** [review]

File: frontend/src/components/SearchBar.tsx
Comment (JoachimTerepo) [reply frontend/src/components/SearchBar.tsx:1]: These were Hardcoded for testing purposes before the Article Submission were up and running. I can update it to run the articles now.

File: frontend/src/styles/globals.css
Comment (JoachimTerepo) [reply frontend/src/styles/globals.css:1]: Yes, I had issues with referencing /app/global.css so I had to create a separate folder styles/global.css

**Pull request merged**